manifests/prow-test-pods/secretdefinitions/s3-credentials.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: secrets-manager.tuenti.io/v1alpha1
+kind: SecretDefinition
+metadata:
+  name: s3-credentials
+spec:
+  name: s3-credentials
+  keysMap:
+    service-account.json:
+      path: secret/data/kubernetes/pro-base/ocp4/common/3scale-pro-base-prow-s3-credentials
+      key: service-account.json
+
```

Comment:
```suggestion
```

Reply:
New line removed in https://github.com/3scale-ops/test-infra/pull/1/commits/d91290a2745388e061ddf8f2117e11bad9c4ebe9